client/my-sites/site-settings/form-security.jsx

```diff
@@ -73,23 +98,63 @@ class SiteSettingsFormSecurity extends Component {
 				<QueryJetpackSettings siteId={ siteId } />
 
 				{ ! isAtomic && (
-					<div>
-						<SettingsSectionHeader
-							disabled={ isRequestingSettings || isSavingSettings || disableSpamFiltering }
-							isSaving={ isSavingSettings }
-							onButtonClick={ handleSubmitForm }
-							showButton
-							title={ translate( 'Jetpack Anti-spam' ) }
-						/>
-						<SpamFilteringSettings
-							dirtyFields={ dirtyFields }
-							fields={ fields }
-							currentAkismetKey={ settings.wordpress_api_key }
-							isSavingSettings={ isSavingSettings }
-							isRequestingSettings={ isRequestingSettings }
-							onChangeField={ onChangeField }
-						/>
-					</div>
+					<Panel header={ translate( 'Jetpack Anti-spam' ) }>
+						{ ! inTransition && ! hasAkismetFeature && ! isValidKey ? (
+							<PanelBody>
+								<Banner
+									description={ translate(
+										'Automatically remove spam from comments and contact forms.'
+									) }
+									event={ 'calypso_akismet_settings_upgrade_nudge' }
+									feature={ FEATURE_SPAM_AKISMET_PLUS }
+									plan={ PLAN_JETPACK_PERSONAL }
+									title={ translate(
+										'Defend your site against spam! Upgrade to Jetpack Personal.'
+									) }
+								/>
+							</PanelBody>
+						) : (
+							<PanelBody
+								title={ translate( 'Your site is protected from spam' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Your site is protected from spam.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).